test/e2etest/test-config.tfvars.tmpl

```diff
@@ -129,4 +134,4 @@ bitbucket_nfs_requests_memory = "256Mi"
 bitbucket_nfs_limits_cpu      = "0.25"
 bitbucket_nfs_limits_memory   = "256Mi"
 
-bitbucket_termination_grace_period = 0
\ No newline at end of file
+bitbucket_termination_grace_period = 0
```

Comment:
I wonder why your formatting is stripping all empty lines in the `.tf` files. What editor do you use?

Reply:
Can be Atom+Goland+PyCharm :)